align/compiler/write_verilog_lef.py

```diff
@@ -256,7 +256,7 @@ def generate_lef(name, values, available_block_lef,
             convert_to_unit(values)
             size = '_'.join(param+str(values[param]) for param in values)
         block_name = name + '_' + size.replace('.','p')
-        res_unit_size = 300
+        res_unit_size = 600
```

Comment:
How is res_unit_size being determined? Is this technology specific? Is it a user-defined input?

Reply:
This number ensures the height of resistance cells remains same as the default height of transistor cells. As the cells have dummies included in it now, the cell height increased thus changed this number. We can modify it such that I pass only the transistor fins and cell generator can translate that information to the resistance height.

Reply:
We can leave this for a different PR as it will need changes from @arvuce22 as well. Please turn it into an issue. We can go ahead and merge this PR.